index.html

```diff
@@ -172,5 +172,21 @@ <h3 id="about-myself">About <br> Myself</h3>
       </div>
      </div>
   </section>
+  <!-- Contact form -->
+  <section class="contact">
+    <div class="contact-box">
+      <h2 class="contact-title">Contact me</h2>
+      <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
+      <form action="https://formspree.io/f/mrgvqqne" method="post">
+        <input type="text" name="client_name" placeholder="Yeremias" required maxlength="30">
+        <input type="email" name="client_mail" placeholder="natajayanj14@gmail.com" required>
+        <textarea name="client_msg" cols="30" rows="6" required maxlength="500">Write your message here</textarea>
+        <fieldset>
+          <button type="button" id="submit">Get in touch</button>
```

Comment:
- Please, kindly make the `type="submit"` because you've changed the type to an ordinary button. if the type weren't specified by you, it would default to submit because it's a button in a form. please, fix this issue with the submit button to make your form submit as now, it won't get submited.

Reply:
Ok. Thanks